sdk/identity/Azure.Identity/tests/IdentityTestEnvironment.cs

```diff
@@ -31,6 +31,7 @@ public class IdentityTestEnvironment : TestEnvironment
 
         public string TestPassword => GetOptionalVariable("AZURE_IDENTITY_TEST_PASSWORD") ?? "SANITIZED";
         public string TestTenantId => GetRecordedOptionalVariable("TENANT_ID") ?? GetRecordedVariable("AZURE_IDENTITY_TEST_TENANTID");
+        public string KeyvaultScope => GetOptionalVariable("AZURE_KEYVAULT_SCOPE") ?? "https://vault.azure.net/.default";
```

Comment:
Yeah, please adding AZURE_KEYVAULT_SCOPE to the public subscription config.

Reply:
Updated.

Reply:
If removing ?? "https://vault.azure.net/.default", some tests will fail. Because when running on Public cloud,  now KeyvaultScope and AuthorityHostUrl  is null leading to tests failed. The reason is AZURE_KEYVAULT_SCOPE and AZURE_AUTHORITY_HOST are not in the ARM template config and live test config.

Reply:
It's in the config, so it's odd that it's now showing up. But I think it's fine to have the default in a common place here regardless for local testing.

Reply:
@pakrym pointed out that this value gets defaulted even outside of the sub config: https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/New-TestResources.ps1#L492, so I don't think we need the default here at all, but am not sure why it's showing empty for you.

Reply:
In addition this variable is used in the recorded test as an input so we might need to use the GetRecordedVariable

Reply:
See https://dev.azure.com/azure-sdk/public/_build/results?buildId=828673&view=results, if removing it tests will fail. Also regardless for local testing ,so I change to have the default in a common place here.

Reply:
See https://dev.azure.com/azure-sdk/public/_build/results?buildId=828673&view=results, if removing it tests will fail. Also regardless for local testing ,so I change to have the default in a common place here.